src/main/kotlin/at/bitfire/dav4jvm/property/GetETag.kt

```diff
@@ -13,35 +13,44 @@ import at.bitfire.dav4jvm.XmlUtils
 import okhttp3.Response
 import org.xmlpull.v1.XmlPullParser
 
+/**
+ * The GetETag property.
+ *
+ * Can also be used to parse ETags from HTTP responses – just pass the raw ETag
+ * header value to the constructor and then use [eTag] and [weak].
+ */
```

Comment:
See [a47f319](https://github.com/bitfireAT/dav4jvm/pull/8/commits/a47f319d1893f12246fa75341219f06b151d6dca)

Reply:
@tobiasKaminsky Thanks for raising a discussion about this implicit decision.

Currently, there may be null values when no text element is passed at all (see [this test](https://github.com/bitfireAT/dav4jvm/blob/fad3fbd539aebd67817ef3f84ba873bbcab70f30/src/test/kotlin/at/bitfire/dav4jvm/property/GetETagTest.kt#L12) for a sample case). As soon as an ETag is passed as a text element, it will be true or false.

We could also throw an exception in the GetETag constructor, but then an invalid ETag (like in the test) could break parsing the whole Multistatus response. I'll have to check that and will create a new issue then. In such a case, the `<getetag/>` should better be ignored completely; then it could guarantee that `eTag` and `weak` always have a value.